common/mercenaries/wc_mercenaries.txt

```diff
@@ -4,180 +4,178 @@
+	light_cavalry = 600
+	elite_infantry = 50
 }
 hemets_hunters_composition = {
 	levy_size = 1
-	rifle_infantry = 1000
+	archers = 1000
 }
 bloodsail_pirates_composition = {
 	levy_size = 1
-	artillery = 100
-	light_infantry = 700
+	siege_machines = 100
+	heavy_infantry = 600
 	archers = 300
+	# galleys = 40
 }
 dogs_of_war_composition = {
 	levy_size = 1
-	artillery = 50
+	siege_machines = 50
 	heavy_infantry = 500
-	knights = 150 
-	archers = 300
+	light_cavalry = 200
+	archers = 250
 }
 stormriggers_composition = {
 	levy_size = 1
-	rifle_infantry = 100
-	light_infantry = 500
-	archers = 150
+	heavy_infantry = 500
+	archers = 250
 }
 highborne_company_composition = {
 	levy_size = 1
 	heavy_infantry = 500
-	casters = 200
+	light_infantry = 200
 }
 centaur_composition = {
 	levy_size = 1
 	light_cavalry = 500
-	knights = 100
-	horse_archers = 300
+	large_cavalry = 75
+	archers = 300
 }
 troll_composition = {
 	levy_size = 1
-	light_infantry = 700
+	heavy_infantry = 700
 	archers = 300
-	casters = 50
+	light_infantry = 50
 }
 korkron_composition = {
 	levy_size = 2
 	heavy_infantry = 650
-	knights = 100
+	elite_infantry = 75
 	light_cavalry = 250
 }
 taunka_composition = {
 	levy_size = 1
-	light_infantry = 300
-	heavy_infantry = 500
+	heavy_infantry = 700
 	archers = 100
-	casters = 100
+	light_infantry = 100
 }
 vrykul_composition = {
 	levy_size = 3
-	light_infantry = 200
-	heavy_infantry = 500
-	archers = 200
+	pikemen = 200
+	heavy_infantry = 250
+	archers = 250
 	light_cavalry = 100
 	# galleys = 80
 }
 frost_vrykul_composition = {
 	levy_size = 1.5
 	heavy_infantry = 500
-	knights = 200
+	pikemen = 200
 	archers = 300
 }
 wolvar_composition = {
 	levy_size = 1.5
-	light_infantry = 800
-	archers = 200
+	heavy_infantry = 450
+	archers = 100
 }
 gorloc_composition = {
 	levy_size = 2
-	light_infantry = 700
-	archers = 200
-	casters = 100
+	heavy_infantry = 500
+	light_infantry = 50
 }
 tuskarr_composition = {
 	levy_size = 1
-	light_infantry = 400
-	pikemen = 200
-	light_cavalry = 200
+	heavy_infantry = 350
+	light_cavalry = 150
 	archers = 100
-	casters = 100
 }
 northern_composition = {
 	levy_size = 2
-	light_infantry = 300
-	heavy_infantry = 300
+	heavy_infantry = 500
 	light_cavalry = 100
-	archers = 300
+	archers = 400
 }
 goblin_composition = {
 	levy_size = 1
-	light_infantry = 200
-	heavy_infantry = 300
-	pikemen = 400
+	archers = 100
+	heavy_infantry = 200
+	knights = 150
 	war_machines = 100
 }
 snobold_composition = {
 	levy_size = 1
-	light_infantry = 700
+	heavy_infantry = 700
 	archers = 200
-	casters = 100
+	light_infantry = 100
 }
 naval_merc_composition = {
 	levy_size = 9
 	galleys = 10
 }
 cenarion_hold_composition = {
 	levy_size = 4
-	light_infantry = 250
-	heavy_infantry = 500
-	casters = 100
+	heavy_infantry = 350
+	light_infantry = 100
 	archers = 150
+	animals = 250
 }
 mage_composition = {
 	levy_size = 2
-	light_infantry = 100
-	pikemen = 500
-	casters = 400
+	light_infantry = 800
+	archmage = 50
```

Comment:
From 50% casters to 100% casters?
Maybe you should check hired troops compositions.

Reply:
Resolved on https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/731